docs/typescript.md

```diff
@@ -0,0 +1,149 @@
+---
+permalink: /typescript
+title: TypeScript
+---
+
+# Typescript
+
+CodeceptJS supports [type declaration](https://github.com/codecept-js/CodeceptJS/tree/master/typings) for [TypeScript](https://www.typescriptlang.org/). It means that allow you to write your tests in TypeScript. Also, all of your custom steps could be write in TypeScript.
+
+# Why TypeScript?
+
+With the TypeScript writing CodeceptJS tests becomes much easier. If correctly configured TypeScript in your project and correctly configured your IDE, then you can get the following features as result:
```

Comment:
If you configure TS properly in your project as well as your IDE, you will get the following features: